docs/developers/backend/file-uploads.rst

```diff
@@ -0,0 +1,41 @@
+.. _developers_backend_file_uploads:
+
+============
+File Uploads
+============
+
+Upload flow
+===========
+
+This is what happens when files are uploaded through the file upload component of a form:
+
+- The user adds a file to the component. A ``POST`` request is made to ``/api/v1/formio/fileupload`` with the content
+  of the file. This creates an instance of the :class:`TemporaryFileUpload` model. The endpoint
```

Comment:
```suggestion
  of the file. This creates an instance of the :class:`openforms.submissions.models.TemporaryFileUpload` model. The endpoint
```
you need to provide the full python paths to get clickable links to the classes when they get added to the docs.